Swift Code Challenge/Views/UserDetailsView.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  UserDetailsView.swift
+//  Swift Code Challenge
+//
+//  Created by Luiz Eduardo Mello dos Reis on 30/06/22.
+//
+
+import SwiftUI
+import MessageUI
+
+struct UserDetailsView: View {
+    @State var user: Person
+    var body: some View {
+        VStack{
+            MapView(lat: user.location.coordinates?.latitude, long: user.location.coordinates?.latitude)
+                .frame(height: 200)
+                .ignoresSafeArea(edges: .top)
+            VStack{
+                CircleImage(userImage: user.picture.large ?? "")
+                
+                contentBody(user: user)
+            }
+            .offset(y: -200)
+            .padding(.bottom, -130)
+            
+            Spacer()
+            
+            Text("made by @luizmello.dev - click me!")
+                .onTapGesture {
+                    openInstagram(instagramHandle: "luizmello.dev")
+                }
+                .padding(.vertical)
+        }
+    }
+    func openInstagram(instagramHandle: String) {
+        guard let url = URL(string: "https://instagram.com/\(instagramHandle)")  else { return }
+        if UIApplication.shared.canOpenURL(url) {
+            if #available(iOS 10.0, *) {
+                UIApplication.shared.open(url, options: [:], completionHandler: nil)
+            } else {
+                UIApplication.shared.openURL(url)
+            }
+        }
+    }
+}
+
+struct contentBody: View {
```

Comment:
Since this is specific to a `User` object, this should be located elsewhere or named differently but why couldn't this be located in the `body` of `UserDetailsView`?